### ISSUE TITLE
AR + GPS Location Asset questions

### ISSUE THREAD

**Issue by XToGo** [dmbfm/unity-ar-gps-location-issues#37]
Hi!

I'm going to buy your AR + GPS Location asset within the next few days and have some questions. It would be really nice if you could help me out.
1. I've read the guide but couldn't find anything for getting the players coordinates so that I can work with them to place prefabs at the same location or near him. Is there a method or function for that?
2. Do you know if your asset is compatible with this one for making great maps? https://assetstore.unity.com/packages/tools/integration/go-map-3d-map-for-ar-gaming-68889
3. I'll make the game for android but it's planned to release it for iOS later. Is there anything I need to do to export this for iOS if I focused on android? Of cause the normal unity stuff, but anything regarding your asset?
4. Is there a minimum required api level for android?

Thank you very much in advance for your answer!

**Comment by dmbfm** [comment]
1.  To get the current player location, you can either subscribe to the `OnRawLocationUpdated` or `OnLocationUpdated` events in the `ARLocationProvider` component  (Raw location is unfiltered data which updates more frequently), or access the variable `ARLocationProvider.provider.CurrentLocationRaw`.  

2. I'm not sure. In principle I guess they should not be incompatible. Maybe watchout for their Location Manager( https://gomap-asset.com/docs/go-map-documentation/documentation/location-manager)... since both of the plugins use location services, the settings in one may interfere with the settings in the other. But then again I've never tested this asset so I can't be sure.

3.  The workflow should be exactly the same for android and iOS.

4. Only the API required for AR Foundation and/or Vuforia. For AR Foundation I think the minimum level is 25 (7.1)

**Comment by XToGo** [comment]
Thank you very much! I've purchased the asset and it works great.

But there's another question I got. If I understood this correctly I can create a list of locations where I want a specific prefab to be placed. When does it place the prefab there? Always or only when the player is in a specific range? I wouldn't be so great if I'd make a large list and the prefab would be instantiated at every location even when the player is miles away. Also can I somehow import this data in the map editor to see where I've placed locations to edit them?

**Comment by dmbfm** [comment]
Yes, currently all components place the objects right as they are initialized.

I will add as a feature request the option to set a radius for initialization, it seems like a 
good idea!

If you use the web editor (still experimental, but functional) you can save and load the 
data to edit it later. Here is a video showing how it works currently: https://youtu.be/7-WCEVuSkK8
